routes/github/contributors.js

```diff
@@ -0,0 +1,68 @@
+// @ts-check
+const { getContributors } = require("respec-github-apis/contributors");
+const { getUsersDetails } = require("respec-github-apis/users");
+const { TTLCache } = require("respec-github-apis/utils/cache");
+
+const CACHE_TTL = 3 * 24 * 60 * 60 * 1000; // 3 days
```

Comment:
3 days seems enough. Can change.

Reply:
Yeah, let’s see how it goes. I don’t imagine people using this a lot (adds vanity noise to the spec).